plugins/outputs/opentsdb/opentsdb.go

```diff
@@ -28,27 +30,43 @@ var sampleConfig = `
   ## prefix for metrics keys
   prefix = "my.specific.prefix."
 
-  ## Telnet Mode ##
-  ## DNS name of the OpenTSDB server in telnet mode
+  ## DNS name of the OpenTSDB server
+  ## Using "opentsdb.example.com" or "tcp://opentsdb.example.com" will use the
+  ## telnet API. "http://opentsdb.example.com" will use the Http API.
   host = "opentsdb.example.com"
 
-  ## Port of the OpenTSDB server in telnet mode
+  ## Port of the OpenTSDB server
   port = 4242
 
+  ## Number of data points to send to OpenTSDB in Http requests.
+  ## Not used with telnet API.
+  httpBatchSize = 50
+
   ## Debug true - Prints OpenTSDB communication
   debug = false
 `
 
-type MetricLine struct {
-	Metric    string
-	Timestamp int64
-	Value     string
-	Tags      string
+type TagSet map[string]string
```

Comment:
why did you make this type? just using a map[string]string is more explicit and readable throughout